ca/certificate-authority.go

```diff
@@ -156,7 +121,15 @@ func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config
 		return nil, err
 	}
 
-	pa := policy.NewPolicyAuthorityImpl()
+	dbMap, err := sa.NewDbMap(paConfig.DBConnect)
+	if err != nil {
+		return nil, err
+	}
+
+	pa, err := policy.NewPolicyAuthorityImpl(dbMap, paConfig.EnforcePolicyWhitelist)
```

Comment:
So isn't the answer to have the PA do its own cleanup, and pass the PA into the CA/RA ctors, as @jmhodges suggests?